apps/discovery_api/priv/static/tableau/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "connector.js",
   "scripts": {
     "test": "jest",
-    "es-check": "es-check es5 connector.js"
+    "es-check": "es-check es5 connector.js connector.config.js"
```

Comment:
This should include auth0-wrapper.js as well.  Look to see if there's a better way to only check production code (excluding our test files).